source/aggregation.txt

```diff
@@ -80,6 +80,40 @@ The following limitations apply when using aggregation operations:
    </reference/operator/aggregation/graphLookup/>` stage has a strict
    memory limit of 100 megabytes and ignores the ``allowDiskUse`` option.
 
+.. _kotlin-sync-atlas-search-stage:
+
+Pipelines Stages for Atlas Search
+---------------------------------
+
+:atlas:`Atlas Search </atlas-search>` queries take the form of an aggregation pipeline stage. Atlas
+Search provides ``$search`` and ``$searchMeta`` stages, both of which must be the first
+stage in any query pipeline. For more information about Atlas pipeline stages,
+see the :atlas:`Choose the Aggregation Pipeline Stage
+</atlas-search/query-syntax/>` page in the Atlas
+manual. 
+
+.. sharedinclude:: dbx/jvm/atlas-search-operator-helpers.rst
+
+   .. replacement:: atlas-query-operators-example
```

Comment:
S: in the docs-shared file - change the code description to actually reference the strings as they appear in the text. This is helpful for making sure users see the 1:1 mapping of the description to the code example
i.e
The following code creates a search stage for a pipeline with the following filters:

- ``genres`` array field includes ``"drama"``

etc